scripts/auto_sshfs_mount.py

```diff
@@ -29,7 +29,7 @@ def Mount(local, remote, ServerAliveInterval=20, ServerAliveCountMax=3):
         raise(e)
     
     
-def AttemptUnmount(local):
+def _AttemptUnmount(local):
```

Comment:
As with `_mount` this should be `_attempt_unmount`, and probably doesn't need to be private?